README.md

```diff
@@ -137,9 +137,9 @@ $dropdownBgDarkColor
 
 You can also find an example in our [playground's `palette.styl` file](./playground/.vuepress/styles/palette.styl)
 
-## Compatibility with MeiliSearch
+## Compatibility with Meilisearch
 
-This package only guarantees the compatibility with the [version v0.24.0 of MeiliSearch](https://github.com/meilisearch/MeiliSearch/releases/tag/v0.24.0).
+This package only guarantees the compatibility with the [version v0.24.0 of Meilisearch](https://github.com/meilisearch/Meilisearch/releases/tag/v0.24.0).
```

Comment:
```suggestion
This package only guarantees compatibility with the [version v0.24.0 of Meilisearch](https://github.com/meilisearch/meilisearch/releases/tag/v0.24.0).
```

Reply:
Unfortunately if we change this sentence here we need to change it everywhere as it is used in a regex to automate PR's across all SDK's!